books/free-programming-books-subjects.md

```diff
@@ -246,12 +246,15 @@ Books that cover a specific programming language can be found in the [BY PROGRAM
 
 * [CI/CD for Monorepos: Effectively building, testing, and deploying code with monorepos](https://github.com/semaphoreci/book-monorepo-cicd) - Pablo Tomas Fernandez Zavalia, Marko Anastasov, SemaphoreCI (PDF, EPUB, Kindle)
 * [CI/CD with Docker and Kubernetes Book](https://github.com/semaphoreci/book-cicd-docker-kubernetes) - Marko Anastasov, Jérôme Petazzoni, Pablo Tom F. Zavalia, SemaphoreCI (PDF, EPUB, Kindle)
+* [Container Networking Docker Kubernetes]([https://www.oreilly.com/library/view/container-networking/9781492036845/](https://pdfcoffee.com/qdownload/container-networking-docker-kubernetes-pdf-free.html)) - Michael Housenblas (HTML)
```

Comment:
Remove this. We do not support PDFDrive links. We only allow links from authorized sources